### PR TITLE
Added estimated payout per day :)

### PR DESCRIPTION
**Pull request by Sir-Fancy**
The user stats page was really needing this, so I made a quick edit.

### CODE REVIEW

File: userstats.php [hunk 1]
```diff
@@ -596,6 +597,15 @@
 	print "</span>";
 }
 
+$current_reward = 25; #CHANGE THIS IN 2017!
+$magic_number = 1000000000 * $current_reward * 86400 / 2^32; #hashes from gigahash * reward * seconds in 24 hours / 2^32
+$coins_per_day = $magic_number * $twelve_hour_hashrate / $netdiff;
+
+print "<B>Estimated Payout Per Day</B>";
```
Comment (LinusU) [inline userstats.php:604]: Please, please, please **don't** use uppercase in tag-names. The correct syntax is: `<b>Estimated Payout Per Day</b>`

Comment (Sir-Fancy) [inline userstats.php:604]: Hey, it hurt me too. I was just following the rest of the file's code style.

Comment (LinusU) [inline userstats.php:604]: Ohh, didn't notice. However, it seems to be mixed throughout the file, the nearest lines has `</span>` and `<div>` respectively so it wouldn't hurt to change this line to be lowercase as well.

**Comment by Sir-Fancy** [comment]
I wanted to consistent with the other `<B>` tags either way. I'll probably submit a commit that changes all the uppercase tags to lower case ;)

**Comment by LinusU** [comment]
Super! Sorry for nagging you ;)

**Comment by Sir-Fancy** [comment]
Haha :) Besides the conflict with the case, what do you think of the commit itself?

**Comment by mootinator** [comment]
I'd actually like to see this in the API somewhere, so I can finish writing an elgius adapter for my WP plugin at some point: https://github.com/mootinator/BTCGroupMiningWP/tree/master/dzminercoop-btcguild

**Comment by LinusU** [comment]
I like it! Would be great to have it in the api, I'm thinking of making a really simple mobile app just to show current payout and this would be great to have in as well.

**Comment by wizkid057** [comment]
I think you may be pulling the hashrate data from the wrong place. You're also pulling a human readable format.

No need to pull in the hashrate, as the 3 hour rate is already saved earlier in $u16avghash.

I was, however, going to add something like this to the stats anyway.

-wk

**Comment by Sir-Fancy** [comment]
Can I make another pull request so I can contribute? :D

**Comment by wizkid057** [comment]
I'll go ahead and whip up a neater version of this

**Pull request closed without merging**